src/main/java/io/github/przbetkier/tuscan/client/match/FaceitMatchClient.java

```diff
@@ -78,6 +79,14 @@ public FaceitMatchClient(@Qualifier("faceitClient") WebClient faceitClient,
     }
 
     public MatchFullDetailsResponse getMatchDetails(String matchId, String playerId) {
+        return getMatchStats(matchId, playerId)
+                .zipWith(getDemoUrl(matchId))
+                .map(result -> MatchFullDetailsMapper.Companion.map(result.getT1(), playerId, result.getT2()))
+                .block();
```

Comment:
I know that this metod returned MatchFullDetailsResponse in a blocking manner before, but can we make this method consistent and get rid of this .block() instruction too and return Mono<MatchFullDetailsResponse>?

Reply:
Sure thing